lib/cocoapods/target/pod_target.rb

```diff
@@ -404,10 +405,24 @@ def framework_paths
                                     end
                                   end
                                 end
-            FrameworkPaths.new(framework_source, dsym_source, bcsymbolmap_paths)
+            Xcode::FrameworkPaths.new(framework_source, dsym_source, bcsymbolmap_paths)
           end
           if file_accessor.spec.library_specification? && should_build? && build_as_dynamic_framework?
-            frameworks << FrameworkPaths.new(build_product_path('${BUILT_PRODUCTS_DIR}'))
+            frameworks << Xcode::FrameworkPaths.new(build_product_path('${BUILT_PRODUCTS_DIR}'))
+          end
+          hash[file_accessor.spec.name] = frameworks
+        end
+      end
+    end
+
+    # @return [Hash{String=>Array<Xcode::XCFramework>}] The vendored and non vendored xcframeworks this target
```

Comment:
This says "The vendored and non vendored xcframeworks this target" but I think I see in the method there are only `vendored` ones?